### PR TITLE
Optimise space

### PR DESCRIPTION
**Pull request by Delfil**
- Combined compute what to unfold and the unfold.
- Made Unwrap abstract
- Optimised space, minimum amount of nodes show on every zoom level. Minimal amount of nodes is 20 * zoomLevel
- Removed unused & outdated collapse operations.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/CombineWrapper.java [hunk 1]
```diff
@@ -17,7 +23,7 @@
 	 * 
 	 * <p>
 	 * When a class extending this class is constructed, the given list should
-	 * fullfill the following conditions. <br>
+	 * full fill the following conditions. <br>
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/CombineWrapper.java]: Is spelled fulfil :)

**Comment by TomBrouws** [comment]
Looks good!

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/CombineWrapper.java (hunk 1 above)
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/CombineWrapper.java]: Done

**Comment by kwendel** [comment]
looks good

**Pull request merged**